integration-tests/src/test/java/oracle/weblogic/kubernetes/ItLivenessProbeCustomization.java

```diff
@@ -0,0 +1,487 @@
+            managedServerPodName, beforeRestartCount);
+
+        // copy script to pod
+        String destLocation = "/u01/tempFile.txt";
+        assertDoesNotThrow(() -> copyFileToPod(domainNamespace, managedServerPodName, "weblogic-server",
+            tempFile.toPath(), Paths.get(destLocation)),
+            String.format("Failed to copy file %s to pod %s in namespace %s",
+            tempFile, managedServerPodName, domainNamespace));
+        logger.info("File copied to Pod {0} in namespace {1}", managedServerPodName, domainNamespace);
+
+        try {
+          Thread.sleep(10 * 1000);
+        } catch (InterruptedException ie) {
+        // ignore
+        }
+
+        String expectedStr = "Hello World, you have reached server "
+            + CLUSTER_NAME_PREFIX + i + "-" + MANAGED_SERVER_NAME_BASE + j;;
+
+        checkAppNotRunning(
+            domainNamespace,
+            managedServerPodName,
+            "8001",
+            "sample-war/index.jsp",
+            expectedStr);
+
+        checkAppIsRunning(
+            domainNamespace,
+            managedServerPodName,
+            "8001",
+            "sample-war/index.jsp",
+            expectedStr);
+
+        // get the restart count of the container in pod after liveness probe restarts
+        int afterRestartCount = assertDoesNotThrow(() ->
+            getContainerRestartCount(domainNamespace, null, managedServerPodName, null),
+            String.format("Failed to get the restart count of the container from pod {0} in namespace {1}",
+            managedServerPodName, domainNamespace));
+        logger.info("Restart count after liveness probe {0}", afterRestartCount);
+        assertTrue(afterRestartCount - beforeRestartCount == 1,
+            String.format("Liveness probe did not start the container in pod %s in namespace %s",
+            managedServerPodName, domainNamespace));
+      }
+    }
+  }
+
+  /**
+   * Verify the negative test case of customization of liveness probe.
+   * Build model in image with liveness probe custom script for a 2 clusters domain.
+   * Enable "LIVENESS_PROBE_CUSTOM_SCRIPT" while creating domain CR.
+   * After domain is created copy a temp file named "testFile.txt" into tested managed server pods in
+   * both clusters, which is NOT used by custom script to trigger liveness probe.
+   * Verify the the container managed server pods in both clusters are NOT restarted
+   */
+  @Test
+  @Order(2)
+  @DisplayName("Test customization of the liveness probe")
+  public void testLivenessProbeNegative() {
+    Domain domain1 = assertDoesNotThrow(() -> getDomainCustomResource(domainUid, domainNamespace),
+        String.format("getDomainCustomResource failed with ApiException when tried to get domain %s in namespace %s",
+            domainUid, domainNamespace));
+    assertNotNull(domain1, "Got null domain resource");
+
+    // create temp file
+    String fileName = "testFile";
```

Comment:
I think it's not necessary to create this file as it's not used.

Reply:
Makes sense. Removed